drivers/hmis/app/models/hmis/hud/processors/base.rb

```diff
@@ -290,6 +290,28 @@ def normalize_single_custom_field_value(cded, value)
       else
         raise "unexpected value \"#{value}\""
       end
+    when 'file'
+      case value
+      when String
+        # If the value is a string, it should be a blob ID for a file that's just been uploaded to ActiveStorage,
+        # but doesn't yet have a File record in our database.
+        blob = ActiveStorage::Blob.find_by(id: value)
```

Comment:
Should we check to see whether this blob id is already persisted? (Maybe that's not expected, but it still seems like something we should handle gracefully)

And can we add a specific error for the case where the `ActiveStorage::Blob` is not found?